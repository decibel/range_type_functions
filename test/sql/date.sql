\set ECHO none

\set range_type tsrange
\set subtype timestamp
\i test/helpers/type_setup.sql

INSERT INTO test VALUES
    ( 'current_date +/- 1'
      , array[
          current_date
          , current_date + 1
          , current_date + 1
          , current_date + 1
          , current_date - 1
          , current_date - 1
          , current_date - 1
        ]
      , :range_type(
          current_date - 1
          , current_date + 1
          , '[]'
      )
      , :range_type(
          current_date - 1
          , current_date + 1
          , '[)'
      )
    )
;


\i test/helpers/type_run.sql

-- vi: expandtab ts=2 sw=2
