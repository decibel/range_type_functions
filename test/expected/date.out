\set ECHO none
1..2
ok 1 - to_range from current_date +/- 1
ok 2 - current_date +/- 1
# TRANSACTION INTENTIONALLY LEFT OPEN!
